.github/workflows/test.yml

```diff
@@ -431,13 +419,13 @@ jobs:
           artifact: manticore_test_kit.img
           image: test-kit:img
           test_prefix: test/clt-tests/core/
-
+          
```

Comment:
We do not need extra spaces on the empty line, it's a good idea to run trim line endings.